main.js

```diff
@@ -413,3 +418,29 @@ function connect() {
     });
     peerConnections[connection.peer] = connection;
 }
+
+window.addEventListener('mousedown', e => {
```

Comment:
click?

Reply:
I had it as a click originally. The problem with click is when I select the state json with the mouse by starting on the bottom right corner and dragging it to the top left corner. If my movement ends where the mouse pointer is outside of the modal, once the mouse button is released it registers as click event over that area, which causes the modal to disappear. It's probably a bug in the browser, and this was a way to avoid it.